public/index.js

```diff
@@ -1,23 +1,31 @@
 const user = {};
 
-const addChatMessageToDOM = (nickname, message) => {
+const addMessageToDOM = message => {
   const listItem = document.createElement('li');
-  listItem.textContent = `${nickname}: ${message}`;
+  if (message.type == 'chat') {
+    listItem.textContent = `${message.nickname}: ${message.body}`;
+  } else if (message.type == 'system') {
+    listItem.textContent = message.body;
+    listItem.style.fontStyle = 'italic';
+  }
   document.getElementById('messages').appendChild(listItem);
 };
 
 document.addEventListener('DOMContentLoaded', () => {
   const socket = io();
+  let message = {};
 
-  socket.on('system message', message => {
-    const listItem = document.createElement('li');
-    listItem.style.fontStyle = 'italic';
-    listItem.textContent = message;
-    document.getElementById('messages').appendChild(listItem);
+  socket.on('system message', messageBody => {
+    message.type = 'system';
+    message.body = messageBody;
+    addMessageToDOM(message);
   });
 
-  socket.on('chat message', ({ nickname, message }) => {
-    addChatMessageToDOM(nickname, message);
+  socket.on('chat message', ({ nickname, messageBody }) => {
```

Comment:
Idem regarding `messageBody`, you can just call it `body` and it will be fine IMO.